drivers/SmartThings/zigbee-button/profiles/button-profile.yml

```diff
@@ -6,6 +6,8 @@ components:
     version: 1
   - id: battery
     version: 1
+  - id: temperatureMeasurement
+    version: 1
```

Comment:
So we don't want to add this, because, presumably, the devices already using this profile do not support temperature measurement. Instead, you can change the test to use the `one-button-temp-battery` profile that already exists.

Reply:
30ee164ddb4d503fa23b3da7c39ff6fdcb8cd307: removed this change and changed the test to use `one-button-temp-battery`